gcp/appengine/frontend3/src/templates/home.html

```diff
@@ -157,14 +157,15 @@ <h3 class="code-card-title">Query by version number</h3>
         </div>
       </div>
     </div>
-    <div id="use-the-api" class="mdc-layout-grid__cell--span-12 usage-examples">
-      <h2 class="heading">Command line tools</h2>
+    <div id="use-the-cli" class="mdc-layout-grid__cell--span-12 usage-examples">
+      <h2 class="heading">Command Line Tools (OSV-Scanner)</h2>
       <div class="cmdline code-card mdc-layout-grid__cell--span-12 mdc-layout-grid__cell--span-12-tablet">
         <h3 class="code-card-title">Scan an SBOM or Lockfile (preview)</h3>
         <pre class="code-card-content" id="example-sbom-scan">
 go install github.com/google/osv-scanner/cmd/osv-scanner@latest
 $GOPATH/bin/osv-scanner --sbom=cycloned-or-spdx-sbom.json
 $GOPATH/bin/osv-scanner --lockfile=package-lock.json
+$GOPATH/bin/osv-scanner -r path/to/your/project
```

Comment:
Would it make sense to add a "# Recursively scan entire local directory" explanation for this?

Reply:
Separated it into a different box, which might look a bit better.